src/Sylius/Bundle/ApiBundle/EventListener/AuthenticationSuccessListener.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\EventListener;
+
+use Lexik\Bundle\JWTAuthenticationBundle\Event\AuthenticationSuccessEvent;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Core\Model\ShopUserInterface;
+
+final class AuthenticationSuccessListener
+{
+    public function onAuthenticationSuccessResponse(AuthenticationSuccessEvent $event): void
+    {
+        $data = $event->getData();
+        $user = $event->getUser();
+
+        if (!$user instanceof ShopUserInterface) {
+            return;
+        }
+
+        /** @var CustomerInterface $customer */
+        $customer = $user->getCustomer();
+
+        $data['customerId'] = $customer->getId();
```

Comment:
I wonder if we shouldn't return here an IRI for customer instead of id, wdyt @lchrusciel?